tests/console/kdump_and_crash.pm

```diff
@@ -33,20 +33,32 @@ sub run {
     activate_kdump;
 
     # restart to activate kdump
-    power_action('reboot');
+    check_var('BACKEND', 'spvm') ? power_action('reboot', keepconsole => 1) : power_action('reboot');
+    reconnect_mgmt_console if check_var('BACKEND', 'spvm');
     $self->wait_boot;
     select_console 'root-console';
 
+    # WIP: FIX IT
+    systemctl 'status kdump';
+    script_run 'echo $?';
     # often kdump could not be enabled: bsc#1022064
     return 1 unless kdump_is_active;
     do_kdump;
-    power_action('reboot', observe => 1, keepconsole => 1);
+    if (get_var('FADUMP')) {
+        reconnect_mgmt_console;
+        assert_screen 'grub2', 180;
+        send_key 'ret';
+        assert_screen 'fadump-second-reboot', 90;
+    }
+    else {
+        power_action('reboot', observe => 1, keepconsole => 1);
+    }
     # Wait for system's reboot; more time for Hyper-V as it's slow.
     $self->wait_boot(bootloader_time => check_var('VIRSH_VMM_FAMILY', 'hyperv') ? 200 : undef);
     select_console 'root-console';
 
     # all but PPC64LE arch's vmlinux images are gzipped
-    my $suffix = get_var('OFW') ? '' : '.gz';
+    my $suffix = check_var('ARCH', 'ppc64le') ? '' : '.gz';
```

Comment:
I think , `OFW ` var was an agreement to check all possibilities of PPC

Reply:
OFW covers only qemu ppc64le